docs-ref-services/legacy/identity-readme.md

```diff
@@ -373,6 +372,9 @@ Credentials can be chained together to be tried in turn until one succeeds using
 Configuration is attempted in the above order. For example, if values for a client secret and certificate are both present, the client secret will be used.
 
 ## Troubleshooting
+
+### Chaining credentials
```

Comment:
Why we add this line?

Reply:
There's a bookmark on line 295: [chaining credentials](#chaining-credentials)

The bookmark didn't have a heading to go with it, I assume that this is the section of the article the bookmark was trying to go to.  Let me know if you would like this change reverted.  Thanks!